tests/unit/igraph_perfect.c

```diff
@@ -0,0 +1,73 @@
+/* -*- mode: C -*-  */
+/*
+   IGraph library.
+   Copyright (C) 2006-2021  Gabor Csardi <csardi.gabor@gmail.com>
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc.,  51 Franklin Street, Fifth Floor, Boston, MA
+   02110-1301 USA
+
+*/
+
+#include <igraph.h>
+#include "igraph_structural.h"
+
+#include "test_utilities.inc"
+
+int main() {
+    igraph_t graph, comp_graph;
+    igraph_bool_t is_perfect;
+    int x;
+
+    printf("bipartite \n");
+    printf("==========================================================\n");
+    igraph_bipartite_game(&graph, NULL, IGRAPH_ERDOS_RENYI_GNM, 10, 10, 0, 20, 0, IGRAPH_ALL);
+    igraph_is_perfect(&graph, &is_perfect);
+    IGRAPH_ASSERT(is_perfect == 1);
+    igraph_destroy(&graph);
+    x = IGRAPH_FINALLY_STACK_SIZE();
+
+    printf("complanetry to star graph size 10 - chordal\n");
+    printf("==========================================================\n");
+    igraph_star(&graph, 10, IGRAPH_STAR_UNDIRECTED, 0);
+    igraph_complementer(&comp_graph, &graph, 0);
+    igraph_is_perfect(&comp_graph, &is_perfect);
+    IGRAPH_ASSERT(is_perfect == 1);
```

Comment:
Please use `IGRAPH_ASSERT(is_perfect)`. It is good practice not to assume a specific representation for "true".